src/unix/linux_like/linux/musl/mod.rs

```diff
@@ -38,11 +37,72 @@ impl siginfo_t {
             _si_overrun: ::c_int,
             si_value: ::sigval,
         }
-
```

Comment:
The version in `src/unix/linux_like/linux/gnu/mod.rs` does not include these blank lines.